themes/timetree/layouts/partials/head.html

```diff
@@ -1,13 +1,10 @@
 <head>
-    <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <title>{{ block "title" . }}{{ with .Params.Title }}{{ . }} | {{ end }}{{ .Site.Title }}{{ end }}</title>
-
+    <meta name="description" content="{{ .Params.Description | default .Summary | default .Site.Params.description }}">
     {{ $sass := resources.Get "scss/main.scss" }}
-    {{ $style := $sass | resources.ToCSS }}
+    {{ $styleOptions := (dict "targetPath" "main.css" "outputStyle" "compressed" "enableSourceMap" (not hugo.IsProduction)) }}
+    {{ $style := $sass | resources.ToCSS $styleOptions }}
```

Comment:
@rlskoeser what does this do? "{{ $style := $sass | resources.ToCSS $styleOptions }}"

Reply:
it's using the configured options on the line before to compile the scss into css, so it specifies the filename, compression, and generates a sourcemap if it's not a production environment (I should do something similar for the js minification).